pkg/lib/config/identity.go

```diff
@@ -282,7 +282,8 @@ func OAuthSSOProviderConfigSchemaBuilder(builder validation.SchemaBuilder) valid
 		Property("alias", validation.SchemaBuilder{}.Type(validation.TypeString).MinLength(1)).
 		Property("modify_disabled", validation.SchemaBuilder{}.Type(validation.TypeBoolean)).
 		Property("create_disabled", validation.SchemaBuilder{}.Type(validation.TypeBoolean)).
-		Property("delete_disabled", validation.SchemaBuilder{}.Type(validation.TypeBoolean))
+		Property("delete_disabled", validation.SchemaBuilder{}.Type(validation.TypeBoolean)).
+		Property("is_active", validation.SchemaBuilder{}.Type(validation.TypeBoolean))
```

Comment:
According to our naming convention, it should be `disabled` (Note the `d` at the end) instead of `is_active`

It is `disabled` because the zero value `false` means enabled.

Reply:
My concern is disabled looks like the whole item is disabled, but in fact the other configs are still effective. But I don't have other idea so let me name it as disabled first.